cmd/queue/main.go

```diff
@@ -137,6 +142,10 @@ type config struct {
 	ServingRequestMetricsBackend string `split_words:"true" required:"true"`
 	ServingRequestLogTemplate    string `split_words:"true" required:"true"`
 	ServingReadinessProbe        string `split_words:"true" required:"true"`
+	TracingConfigEnable          string `split_words:"true" required:"true" default:"false"`
```

Comment:
Are those _really_ required? If I disable tracing, the rest are redundant.

Reply:
Also please make booleans `bool`, which will remove the need for parsing below.

Reply:
thanks for the tip, I didn't realize this!  Just tried it out, and it works.